lib/linguist/languages.yml

```diff
@@ -4461,6 +4461,20 @@ SRecode Template:
   extensions:
   - ".srt"
   language_id: 335
+SSHConfig:
+  type: data
+  group: INI
+  aliases:
+  - ssh-config
+  - ssh_config
```

Comment:
@pchaigno Doesn't Linguist recognise alias-variants which use dashes or underscores instead of spaces?

If so, we can remove the `aliases` list.

Reply:
I copied from the sources in language-etc, will test without aliases.